docs/development/maintainers.md

```diff
@@ -108,3 +108,15 @@ final number. Follow the relevant steps from {ref}`release-instructions`.
 
 Cherry pick the corresponding documentation commits to the `stable` branch. Use
 `[skip ci]` in the commit message.
+
+### Upgrading pyodide to a new version of CPython
+
+For example: `v3.11.1` -> `v3.11.2`
+1. Download the **Gzipped source tarball** at https://www.python.org/downloads/release/python-3112 into `downloads/`
+2. `shasum -a 256 downloads/Python-3.11.2.tgz > cpython/checksums`
+3. `git grep --name-only "3.11.1" ` # All these files will need to be updated.
+4. After updating then Python version in `Dockerfile`, create a new Docker image.
+5. Upload the new Docker image to https://hub.docker.com/r/pyodide/pyodide-env
```

Comment:
@hoodmane @rth
Could one of you please do steps 4 and 5 using the `Dockerfile` above so that I can complete the process?